redis_hashring/__init__.py

```
@@ -79,21 +108,46 @@ class RingNode(object):
     ```
     """
 
-    def __init__(self, conn, key, n_replicas=RING_REPLICAS):
+    def __init__(
+        self,
+        conn,
+        key,
+        *,
+        n_replicas=RING_REPLICAS,
+        hash_algorithm=HashAlgorithm.XXHASH,
```

Encouraging a better default here is fine with me if you bump the middle version number, especially since it fails early if the package is not installed, but it is a breaking change.